packages/design-tokens/bin/buildCssVarTokens.ts

```diff
@@ -112,6 +112,7 @@ const run = () => {
     path.resolve(jsonOutput, "color.json"),
     formatJson(
       JSON.stringify({
+        dataViz: customPropertiesThemeVersion3.dataViz,
```

Comment:
This was the only source change i made in this commit.
After more research on the stylelint plugin and a pinch of serendipity, i stumbled upon this within murmur: 
![image](https://user-images.githubusercontent.com/7651503/126118236-7a47702c-eb2a-4f94-852f-e7c39cb96e13.png)

Because we are removing `variable-identifiers.scss` in the breaking change, there would have been no way for this file in murmur to be fixed without somehow having access to an alternative.